server/router/auth.ts

```diff
@@ -47,7 +49,7 @@ router.post('/', (req, res) => {
             res.send(err);
           } else {
             const userList = data.Items;
-            res.send({ id: userList![0].id ?? null });
+            res.send({ id: userList?.length ? userList[0].id : null });
```

Comment:
Is it not possible that an empty list can be provided and you will still access index 0?

Reply:
when i was testing it before i couldn't access index 0, but for some reason i was able to when trying to add the dispatcher auth, so ill do some more looking into it and see